aws_adfs/login.py

```diff
@@ -151,7 +155,7 @@ def _authenticate(config, username=None, password=None):
     del password
 
     # Decode the response
-    html = ET.fromstring(response.text.decode('utf8'), ET.HTMLParser())
+    html = ET.fromstring(response.text, ET.HTMLParser())
```

Comment:
@Brandond would you like to exclude this change from pull request. I'll add Travis and couple of test to verify that tool works with different versions of Python. I'll try to find out how to reproduce issue added by @dzinek.